woocommerce/woocommerce-ios
# Introduce `yesterday` range option

Part of #8146

Why
==========
The final part of issue #8146 is to deliver the `Yesterday`, `Last Week`, `Last Month`, `Last Year`, `Last Quarter` and `Quarter to Date` ranges. This one aims to the `Yesterday` range as the first batch of changes.

How
==========
This PR introduces the `Yesterday` range, alongside the respective Range data struct and unit tests. It also makes small adjustments to the range data class naming pattern to prepare for the remaining ranges. 

Screen Capture
==========
https://user-images.githubusercontent.com/5920403/205669227-8c4ce6bb-2a69-4fac-89e0-f65619a78e63.mp4

How to Test
==========
1. Open the Analytics Hub
2. Hit the Date range selection view
3. Verify that the `Yesterday` option is now available there. Click on it.
4. Verify that the date range now compares yesterday with the day before yesterday.

Update release notes:
